### DISCUSSION TITLE
JWT verification with JWKs

### DISCUSSION THREAD

**Discussion by bijayzbyte** [seaweedfs/seaweedfs#5727] [Q&A]
is it possible to Verify JWT Signatures using JWKs in SeaweedFS Filer JWT verification?
if so what are the config filer jwt settings to be done in security-configmap.yaml

**Comment by gtudan** [comment]
I'm interested in this as well - did you find a solution?

**Reply by bijayzbyte** [reply]
couldn't find a solution i think there is not support for it in seaweedfs
but would love to research on it and integrate it in seaweedfs
for now am going with jwt verification only
